.github/workflows/quick-jobs.yml

```diff
@@ -92,3 +92,36 @@ jobs:
         run: make doctest-install
       - name: Doctest
         run: make doctest
+  buildinfo:
+    name: Check buildinfo
```

Comment:
Could the name be a little more descriptive? The “Are buildinfo docs up to date?” below could be a good starting point. Maybe: “Check generation of buildinfo docs (list of Cabal format fields)”.

Reply:
I started with a much longer name but then noticed that it was out of step with the other short names. It was something like "Generate buildinfo documentation and check for diffs".